model/clusters_mgmt/v1/aws_infrastructure_access_state_type.model

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// State of an AWS incfrastructure access role grant.
+enum AWSInfrastructureAccessState {
+	// The access role is in the progress of being granted.
+	InProgress
```

Comment:
In other places when something is in progress we use `pending`. For example, we use `pending` when the cluster is waiting for an account claim. Can we do the same here?

Reply:
Those are the terms aws account operator uses. If we want `pending` this means I need to add a mapper, or our API can differ that our model, and the model can match account operator.
Anyway, I'll change it.